rllib/core/learner/learner.py

```diff
@@ -1294,24 +1295,8 @@ def _update_from_batch_or_episodes(
             if not self.should_module_be_updated(module_id, batch):
                 del batch.policy_batches[module_id]
 
-        # Log all timesteps (env, agent, modules) based on given episodes.
```

Comment:
FInally, this goes away haha.

Reply:
We probably need to remvoe this also from `learn_from_iterator`

Reply:
Great catch. Will check ...

Reply:
done